command/polly_tts.js

```diff
@@ -7,19 +7,23 @@ var glob = require('glob');
 var polly_tts = {
   getPollyChunk: function(text, filenameIndex, audio_file) {
     return new Promise(function(resolve, reject) {
+      let rate = process.env.PROSODY_RATE || 'medium';
```

Comment:
When adding new environment variables, can you also add it to `app.json`? That file is used by Heroku to auto-deploy new builds.